flask_resty/api.py

```diff
@@ -28,6 +27,12 @@ def handle_api_error(error):
     def _get_app(self, app):
         app = app or self._app
         assert app, "no application specified"
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        app.extensions.setdefault('resty', {
```

Comment:
This should be an actual class. See e.g. https://github.com/mitsuhiko/flask-sqlalchemy/blob/master/flask_sqlalchemy/__init__.py#L819.